src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -823,6 +838,32 @@ private Integer stringToIntWithOption(String rawInput, String optionFlag, Intege
     return intValue;
   }
 
+  enum Provider {
```

Comment:
Do we need to define a generic CSV provider corresponding to the output from the Dominion conversion process? Even if it behaves the same as ES&S, it would be weird to call it ES&S when it's actually derived from a rival company's output.